docs/content/preview/drivers-orms/nodejs/postgres-node-driver.md

```diff
@@ -103,19 +103,21 @@ The following table describes the connection parameters required to connect usin
 
 The following is an example connection string for connecting to a YugabyteDB cluster with SSL enabled.
 
-```javascript
+```js
 const connectionString = "postgresql://user:password@localhost:port/database?ssl=true&sslmode=verify-full&sslrootcert=~/.postgresql/root.crt"
 const client = new Client(connectionString);
 client.connect()
 ```
 
 If you created a cluster on [YugabyteDB Managed](https://www.yugabyte.com/managed/), use the cluster credentials and [download the SSL Root certificate](../../../yugabyte-cloud/cloud-connect/connect-applications/).
 
-Refer to [Configure SSL/TLS](../../../reference/drivers/nodejs/postgres-pg-reference/#configure-ssl-tls) for more information on node-postgresql default and supported SSL modes, and examples for setting up your connection strings when using SSL.
+Refer to [Configure SSL/TLS](../../../reference/drivers/nodejs/postgres-pg-reference/#configure-ssl-tls) for more information on node-postgresql default and supported SSL modes, and other examples for setting up your connection strings when using SSL.
 
-### Step 3: Query the YugabyteDB cluster from your application
+### Step 3: Write your application
 
-Create a new JavaScript file called `yb-ysql-helloworld-ssl.js` in your project directory. Copy the following code, which sets up tables and queries the table contents. Replace the connection string `yburl` parameters with the cluster credentials and SSL certificate if required.
+Create a new JavaScript file called `QuickStartApp.js` in your project directory.
+
+Copy the following sample code to set up tables and query the table contents. Replace the connection string `yburl` parameters with the cluster credentials and SSL certificate, if required.
 
 ```js
 var pg = require('pg');
```

Comment:
the connection string in the code that follows doesn't seem to relate to the example above at all?
and "yburl" is where in this code?